src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -75,6 +75,9 @@
 
 namespace grpc_core {
 
+DebugOnlyTraceFlag grpc_trace_subchannel(false,
```

Comment:
Please run `tools/distrib/clang_format_code.sh` under `grpc/` so that the code is formatted. Otherwise, the sanity check will fail.

Reply:
Seems like I need to install docker to run this script?

Reply:
Yes.